README.md

```diff
@@ -14,15 +14,6 @@ A state controller with its own debugger
 * [Cerebral v1 documentation website](http://cerebral-website.herokuapp.com/)
 * [Cerebral v2 beta documentation website](http://www.cerebraljs.com/)
```

Comment:
Should be Cerebral v2 without 'beta' here...

Reply:
Why not remove Cerebral v1.0.2 and call it just Cerebral. Especially as we soon could have Cerebral 3+

Reply:
Changed to this:

* [Current Cerebral (2.x and up)](http://www.cerebraljs.com/)
* [Previous Cerebral (1.x)](http://cerebral-website.herokuapp.com/)